src/createClient.ts

```diff
@@ -460,6 +466,22 @@ export class Client<TDocuments extends PrismicDocument = PrismicDocument> {
 
 	fetchOptions?: RequestInitLike
 
+	/**
+	 * Internal parameters that will be sent with each query.
+	 */
+	private internalDefaultParams: Omit<
+		BuildQueryURLArgs,
+		"ref" | "integrationFieldsRef" | "accessToken" | "routes"
+	> =
+		process.env.NODE_ENV === "development"
+			? {
+					[PRISMIC_DEV_PARAM]: 1,
+					[PRISMIC_CLIENT_VERSION_PARAM]: version,
+				}
+			: {
+					[PRISMIC_CLIENT_VERSION_PARAM]: version,
+				}
```

Comment:
💡 #idea: This seems unnecessary as an instance property. Couldn't it be a constant? Or even better, passed directly to `buildQueryURL()` as arguments? It's simpler and requires less code.

```ts
buildQueryURL(this.endpoint, {
  "x-d": process.env.NODE_ENV === "development" ? 1 : undefined,
  "x-c": version
})
```

Disregard this if we decide to inject the parameters within `buildQueryURL()` directly.

Reply:
Let's inject it directly and keep the client out of that